Packs/FortiSandbox/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Forti Sandbox",
     "description": "Pack contains integration with playbooks to upload file for malware analysis, retrieve the results and get file rating for previously scanned files from FortiSandbox",
     "support": "community",
-    "currentVersion": "1.0.2",
+    "currentVersion": "1.1.0",
```

```suggestion
    "currentVersion": "1.0.3",
```
I think a minor update is not necessary here, let's replace it with a revision update (and also the RN name).